xbmc/guilib/GUIControlGroupList.cpp

```diff
@@ -20,7 +20,7 @@
 
 #include "GUIControlGroupList.h"
 #include "input/Key.h"
-#include "GUIInfoManager.h"
+#include "guiinfo/GUIInfoLabels.h"
```

Comment:
As we the use CGUIInfoLabels heavily in our guilib why not move it there instead of the new `guiinfo` folder.